book_manager.rb

```diff
@@ -0,0 +1,24 @@
+require './book'
+
+class BookManager
+  attr_reader :books
+
+  def initialize
+    @books = []
+  end
+
+  def list_books
+    @books.each do |book|
+      puts "Title: #{book.title}, Author: #{book.author}"
+    end
+  end
+
+  def create_book
+    print 'Title: '
+    title = gets.chomp
+    print 'Author: '
+    author = gets.chomp
+    @books << Book.new(title, author)
+    puts 'Book created successfully'
+  end
+end
```

Comment:
- [ ] it would be better to separate the print and `gets. chomp` to another method and then use it inside the create_book in order to achieve the single responsibility principle the create_book should only have the code to create a book like this : 
```ruby
 def create_book
   // here call the method that will print and ask for inputs
    @books << Book.new(title, author)
    puts 'Book created successfully'
  end
```